reminderapp/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+# Create your models here.
+
+
+class Event(models.Model):
+
+    user = models.CharField(max_length=50, default='Guest')
+    event_name = models.CharField(max_length=100)
+    event_date = models.CharField(max_length=50, default='2023-01-01')
+    event_time = models.CharField(max_length=50, default='12:00')
+    reminder_date = models.CharField(max_length=50, default='2023-01-01')
```

Comment:
Please use proper field
https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.DateField

Also please check if there is a field for time.

Reply:
Actually, there is datetimefield: https://docs.djangoproject.com/en/4.2/ref/models/fields/#datetimefield which handles both.